util/rate_limiter.h

```diff
@@ -61,6 +61,9 @@ class GenericRateLimiter : public RateLimiter {
  private:
   void Refill();
   int64_t CalculateRefillBytesPerPeriod(int64_t rate_bytes_per_sec);
+  uint64_t NowMicrosMonotonic(Env* env) {
+    return env->NowNanos() / std::milli::den;
+  }
```

Comment:
I looked for that place and could not fine one when creating the diff. This is a one-liner function (kinda like a macro). So it's fine to leave it as it is. In the future, if the number of these utility functions grow, we should add some utility files.